### PR TITLE
extract labels

### PR DESCRIPTION
**Pull request by revesansparole**
This merge request to allow svg2rlg tools to preserve the label of groups (used by inkscape as layers)

### CODE REVIEW

**Comment by deeplook** [comment]
Thanks! I'm fine with this if you provide a test case as well.

**Comment by revesansparole** [comment]
I added three simple tests. Please tell me if they match your criteria.

File: tests/test_basic.py [hunk 1]
```diff
@@ -428,6 +428,73 @@ def test_css_stylesheet(self):
...
+                   style="opacity:1;fill:none;fill-opacity:1;stroke:#ff0000;stroke-width:3;stroke-linecap:butt;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;marker-end:url(#Arrow1Send)"
+                   d="M 601.06712,388.63166 H 954.67961"
+                   id="path819" />
+                </g>
+            </svg>
+        ''')))
+        main_group = drawing.contents[0]
+        gr, = main_group.contents
+        assert gr.label == "x_axis"
+
```
Comment (claudep) [inline tests/test_basic.py]: Thanks for the tests. I know testing one thing at a time is good practice, but honestly in this case, I think we could as well put all assertions in the same test, as they are closely related and the source svg is identical.

**Comment by claudep** [comment]
Could you explain somewhere (comment in the code/test or commit message) what advantage we get by preserving ids and labels?

File: tests/test_basic.py (hunk 1 above)
Comment (revesansparole) [reply tests/test_basic.py]: I like the idea of keeping atomic tests so I kept the three functions but I changed the svg code to test different cases

**Comment by PancakeFriday** [comment]
I'm looking for this feature as I want to extract specific SVG layers from an SVG file. Is this going to get merged (only asking since there has been no change since February).

**Comment by claudep** [comment]
Thanks, I squashed and removed formatting changes in #258.

**Pull request closed without merging**